stack2nix/Stack2nix/Stack.hs

```diff
@@ -113,8 +116,11 @@ data Dependency
   -- | Archive ...
   deriving (Show)
 
+-- flags are { pkg -> { flag -> bool } }
+type PackageFlags = HM.HashMap T.Text (HM.HashMap T.Text Bool)
```

Comment:
Can flags be present/absent + true/false? Or can they just be present or absent? If the second case, then this should be
`HashMap Text (Set Text)`.

Reply:
I don't think they can. Cabal also only has a notion of `-flag` and `+flag`.

The `stack` documentation makes be me believe it's got to be bool spec'd.

https://docs.haskellstack.org/en/stable/yaml_configuration/#flags